feature/splash/src/main/java/com/epmedu/animeal/splash/presentation/viewmodel/SplashViewModel.kt

```diff
@@ -64,12 +79,12 @@ internal class SplashViewModel @Inject constructor(
                 }
                 isPhoneNumberVerified -> {
                     setMobileAuthenticationTypeUseCase()
-                    setFinishProfileAsStartDestinationUseCase()
+                    setNextDestinationByNetworkState()
```

Comment:
Why not just check the network state before making a request to get `isPhoneNumberVerified`?